inference/core/workflows/core_steps/transformations/image_slicer/v1.py

```diff
@@ -0,0 +1,216 @@
+the model was trained on. Models generally perform best on data that is similar to what they encountered during 
+training. The default size of slices is 640, but this might not be optimal if the model's input size is 320, as each 
+slice would be downsized by a factor of two during inference. Similarly, if the model's input size is 1280, each slice 
+will be artificially up-scaled. The best setup should be determined experimentally based on the specific data and model 
+you are using.
+
+To learn more about SAHI please visit [Roboflow blog](https://blog.roboflow.com/how-to-use-sahi-to-detect-small-objects/)
+which describes the technique in details, yet not in context of Roboflow workflows.
+"""
+
+
+class BlockManifest(WorkflowBlockManifest):
+    model_config = ConfigDict(
+        json_schema_extra={
+            "name": "Image Slicer",
+            "version": "v1",
+            "short_description": "Splits input image into series of smaller images to perform accurate prediction.",
+            "long_description": LONG_DESCRIPTION,
+            "license": "Apache-2.0",
+            "block_type": "transformation",
+        }
+    )
+    type: Literal["roboflow_core/image_slicer@v1"]
+    image: Union[WorkflowImageSelector, StepOutputImageSelector] = Field(
+        title="Image to slice",
+        description="The input image for this step.",
+        examples=["$inputs.image", "$steps.cropping.crops"],
+        validation_alias=AliasChoices("image", "images"),
+    )
+    slice_width: Union[PositiveInt, WorkflowParameterSelector(kind=[INTEGER_KIND])] = (
+        Field(
+            default=640,
+            description="Width of each slice, in pixels",
+            examples=[320, "$inputs.slice_width"],
+        )
+    )
+    slice_height: Union[PositiveInt, WorkflowParameterSelector(kind=[INTEGER_KIND])] = (
+        Field(
+            default=640,
+            description="Height of each slice, in pixels",
+            examples=[320, "$inputs.slice_height"],
+        )
+    )
+    overlap_ratio_width: Union[
+        Annotated[float, Field(ge=0.0, lt=1.0)],
+        WorkflowParameterSelector(kind=[FLOAT_ZERO_TO_ONE_KIND]),
+    ] = Field(
+        default=0.2,
+        description="Overlap ratio between consecutive slices in the width dimension",
+        examples=[0.2, "$inputs.overlap_ratio_width"],
+    )
+    overlap_ratio_height: Union[
+        Annotated[float, Field(ge=0.0, lt=1.0)],
+        WorkflowParameterSelector(kind=[FLOAT_ZERO_TO_ONE_KIND]),
+    ] = Field(
+        default=0.2,
+        description="Overlap ratio between consecutive slices in the height dimension",
+        examples=[0.2, "$inputs.overlap_ratio_height"],
+    )
+
+    @classmethod
+    def get_output_dimensionality_offset(cls) -> int:
+        return 1
+
+    @classmethod
+    def describe_outputs(cls) -> List[OutputDefinition]:
+        return [
+            OutputDefinition(name="crops", kind=[BATCH_OF_IMAGES_KIND]),
```

Comment:
I think output should be named `slices` instead of crops given other naming on block and properties

Reply:
👍